main.go

```diff
@@ -402,7 +402,7 @@ Flags:
   -o, --out		Specify file to write to. If empty, writes to stdout
   -f, --format		Specify output format. Inferred by --out flag if present and --format flag empty
   -r, --rows		Number of rows to generate
-  -c, --columns 	Number of columns to generate
+  -c, --cols 	Number of columns to generate
```

Comment:
Could you use tabs to make this line up with the rest? You may not see the difference in your editor but you can see the difference in Github's UI.

Reply:
Oh you're right there was a space character there instead of a tab.